tests/all/unit/api/validation/test_createdat.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+#improting datetime
+from datetime import datetime
+from unittest import TestCase
+#importing all models containing createdat variable
+from app.models.user import User
+from app.models.event import Event
+from app.models.access_code import AccessCode
+from app.models.order import Order
+from app.models.session import Session
+from app.models.ticket_holder import TicketHolder
+from app.models.role_invite import RoleInvite
+from app.models.discount_code import DiscountCode
+from app.models.user_token_blacklist import UserTokenBlackListTime
+from app.models.event_invoice import EventInvoice
+
+class TestCreatedatValidation(TestCase):
+
+    def test_createdat_all_models(self):
+        ''' createdat validate time : Tests created_at variable in all models 
+        '''
+        #all the models having created_at are stored in a set 
+        created_at_models=[Session, User, Event , AccessCode, Order, Session, TicketHolder, 
+                            RoleInvite, DiscountCode, UserTokenBlackListTime, EventInvoice]
+        for model in created_at_models:
+            #looping through each model in a subtest to check if created_at time is set to current time
+            with self.subTest(model=model):
+                self.assertEqual(model.created_at,datetime.now(),'\nTests created_at in model {}'.format(model))
```

Comment:
Are you trying to test for the default attribute of created_at in each model here? If so, 
you'll need to instantiate the Alchemy object and then assert its attribute.

Reply:
I am testing already instantiated attribute of created_at in each model to check if it is set to the current timezone. Yeah, instantiating objects and asserting its attribute can also be done if preferred.

Reply:
It's not about preference. This'll never work as you can see in the build failure. It's like saying Session.created_at == data. Session is a class, not an object/instance of session so it has no created_at attribute at all

Recommend running tests locally to verify that they pass and what you're doing is correct before sending in PR

Reply:
Do we need to retrieve the value from db and check or if there is any other way.
After creating an object is there any method returning the object with loaded attributes.
As direct instantiation of object created_at value is stored as None type, as the default is func.now() in class definition of model classes.

Reply:
> Do we need to retrieve the value from db and check

Yes